monitoring/nethealth.go

```diff
@@ -127,6 +127,15 @@ func (c *nethealthChecker) Check(ctx context.Context, reporter health.Reporter)
 }
 
 func (c *nethealthChecker) check(ctx context.Context, reporter health.Reporter) error {
+	peers, err := c.getPeers()
+	if err != nil {
+		log.Debug("Failed to discover nethealth peers.")
```

Comment:
Even if you could just embed the error message - it would still be better than without:
```go
log.Debugf("Failed to discover nethealth peers: %v.", err)
```